### ISSUE TITLE
Using Julia's package manager on Linux requires Git to be installed

### ISSUE THREAD

**Issue by tkelman** [docker-library/julia#1]
This would make the images larger and can easily be done manually by anyone who wants to use the image, so it should at least be noted in the readme if not installed by default. Try running `julia -e 'Pkg.init()'` or `julia -e 'Pkg.add("JSON")'` for example.

Within another few minor versions we should be rewriting the package manager to use LibGit2 and including that in the binaries so this will no longer be necessary. In the meantime figured it was worth opening this.

**Comment by jpfairbanks** [comment]
I ran into this today. I just added `RUN apt-get install -y git` to my `Dockerfile` and use `RUN Pkg.add("PackagName")` in order to use the julia package manager with docker caching.

Is that how this docker image should be used? An alternative is to have the `.julia/v0.3/` directory already set up and copy it into the docker. My intuition is that installing git in the image and then using `RUN` to install packages is the right way to do it, but I am not sure.
